src/main/frontend/state.cljs

```diff
@@ -1545,11 +1545,11 @@ Similar to re-frame subscriptions"
     (if-let [tldraw-app (active-tldraw-app)]
       (let [last-time (:block/updated-at whiteboard-page)
             now (util/time-ms)
-            ellapsed (- now last-time)
```

Comment:
@tiensonqin Is it deliberate or just typo?

Reply:
I think it's a typo.